docs/conf.py

```diff
@@ -11,7 +11,12 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import sys, os
+import sys
+import os
+import os.path
+
+# Use the currently checked out folium of this folder
+sys.path.insert(0, os.path.join(os.pardir))
```

Comment:
`os.path.join()` is doing nothing here. Did you mean `os.path.abspath()`?

Reply:
No, I was just sleeping. It can be removed.